index.js

```diff
@@ -60,15 +61,17 @@ const applyErrorMiddlewares = (app, config) => {
   }));
 };
 
-const getContentSecurityPolicy = config => {
+const getContentSecurityPolicy = (config, res) => {
   let csp = config.csp;
+
+  /* eslint-disable quotes */
   let directives = {
-    /* eslint-disable quotes */
     defaultSrc: ["'none'"],
     styleSrc: ["'self'"],
     imgSrc: ["'self'"],
     fontSrc: ["'self'", 'data:'],
-    scriptSrc: ["'self'", "'unsafe-inline'"]
+    scriptSrc: ["'self'", `'nonce-${res.locals.nonce}'`],
+    'frame-ancestors': ["'none'"]
     /* eslint-enable quotes */
```

Comment:
Can we move this outside the `directives` declaration, like you've done with the disable line above?